src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -345,28 +352,15 @@ export class GKEClusterSettingsComponent
       });
   }
 
-  private _getGKEKubernetesVersions(): void {
+  private _getGKEKubernetesVersions(zone: string, mode: string): void {
     this.kubernetesVersions = [];
-    let zone = this.controlValue(Controls.Zone);
-    let mode = this.controlValue(Controls.KubernetesVersionMode);
     let releaseChannel: string;
 
-    if (!zone) {
-      zone = this.control(Controls.Zone).setValue(this.ZONE_DEFAULT_VALUE);
-    }
-
-    if (!mode) {
-      mode = this.control(Controls.KubernetesVersionMode).setValue(this.KubernetesVersionMode.StaticVersion);
-    }
-
-    if (!this.controlValue(Controls.ReleaseChannelOptions)) {
```

Comment:
IMO, this was not needed because of bug fix https://github.com/kubermatic/dashboard/pull/4975